sdks/java/extensions/zetasketch/src/main/java/org/apache/beam/sdk/extensions/zetasketch/HllCountMergePartialFn.java

```diff
@@ -57,6 +57,7 @@ private HllCountMergePartialFn() {}
   @Override
   public HyperLogLogPlusPlus<HllT> addInput(
       @Nullable HyperLogLogPlusPlus<HllT> accumulator, byte[] input) {
+    if (input == null) return accumulator;
```

Comment:
Also mark output as nullable now. 
Can we deal with a null output of addInput(..) throughout?